### DISCUSSION TITLE
Query about pull requests.

### DISCUSSION THREAD

**Discussion by MinusRF** [twbs/bootstrap#31528] [General]
Greetings, We ( group of 11 members ) have a software development lab in which we require to work as a team and contribute to a live open source project. We need to report  instructor with our progress every week, so if  our codes clear your tests will you accept our pull requests ?

**Comment by patrickhlauke** [comment]
only if the changes/additions are deemed useful and relevant - just passing automated tests is of course not sufficient. ideally the PRs would address discussed open issues.
